test/integration_test_helper.rb

```diff
@@ -8,7 +8,12 @@
 require 'show_me_the_cookies'
 require 'database_cleaner'
 require 'active_support_test_case_helper'
-require 'minitest-optional_retry'
+require 'minitest/retry'
+Minitest::Retry.use!
+
+Minitest::Retry.on_consistent_failure do |klass, test_name|
+  Rails.logger.warn("DO NOT IGNORE - Consistent failure - #{klass} #{test_name}")
```

Comment:
Should this be ```.error```? Warnings tend to be ignored.

Reply:
Just changed it to `.error`

Reply:
Doesn't show up on github for me?